frontend/apps/reader/modules/readerflipping.lua

```diff
@@ -83,22 +74,34 @@ function ReaderFlipping:onSetStatusLine()
 end
 
 function ReaderFlipping:paintTo(bb, x, y)
-    if self.ui.highlight.select_mode then
-        if self[1][1] ~= self.select_mode_widget then
-            self[1][1] = self.select_mode_widget
-        end
+    local widget
+    if self.ui.paging and self.view.flipping_visible then
+        -- pdf page flipping or bookmark browsing mode
+        local icon_size = Screen:scaleBySize(32)
+        widget = IconWidget:new{
```

Comment:
I don't think this change is a good idea: you'll be creating/trashing IconWidgets on each paintTo() !
Before, we were re-using them.

Reply:
It was a dilemma to have them all in `init()` (using memory) or to recreate in `paintTo()`.

Reply:
I think in all cases, the image/blitbuffer themselves will be cached (by ImageWidget) and kept in memory. These IconWidgets are then just small Lua tables.
But it's still best (I think) to keep them alive, rather than to create/imageWidget'ing them/lookup images cache/trashing Lua tables/having them garbage collected.